nl/mail/chrome/messenger/aboutSupportMail.dtd

```diff
@@ -1,26 +1,26 @@
-<!ENTITY aboutSupport.appBasicsAppBuildID "Application Build ID">
+<!ENTITY aboutSupport.appBasicsAppBuildID "Applicatie Build ID">
 
-<!ENTITY aboutSupport.accountsTitle "Mail and News Accounts">
-<!ENTITY aboutSupport.showPrivateData.mainText "Include account names">
-<!ENTITY aboutSupport.showPrivateData.explanationText "(possibly identifying information)">
+<!ENTITY aboutSupport.accountsTitle "Mail en Nieuws Accounts">
+<!ENTITY aboutSupport.showPrivateData.mainText "Inclusief account namen">
+<!ENTITY aboutSupport.showPrivateData.explanationText "(identificeert mogelijk informatie)">
 <!ENTITY aboutSupport.accountsID "ID">
-<!ENTITY aboutSupport.accountsName "Name">
-<!ENTITY aboutSupport.accountsIncomingServer "Incoming server">
-<!ENTITY aboutSupport.accountsOutgoingServers "Outgoing servers">
-<!ENTITY aboutSupport.accountsServerName "Name">
-<!ENTITY aboutSupport.accountsConnSecurity "Connection security">
-<!ENTITY aboutSupport.accountsAuthMethod "Authentication method">
-<!ENTITY aboutSupport.filterCount "Filter Count">
-<!ENTITY aboutSupport.accountsDefault "Default?">
+<!ENTITY aboutSupport.accountsName "Naam">
+<!ENTITY aboutSupport.accountsIncomingServer "Inkomende server">
+<!ENTITY aboutSupport.accountsOutgoingServers "Uitgaande servers">
+<!ENTITY aboutSupport.accountsServerName "Naam">
+<!ENTITY aboutSupport.accountsConnSecurity "Verbinding beveiliging">
+<!ENTITY aboutSupport.accountsAuthMethod "Autenticatie methode">
```

Comment:
CORRECTION NEEDED, it should be: ''Authenticatiemethode''

It's Authenticatie vs. Autenticatie.